ckanext/canada/helpers.py

```diff
@@ -299,6 +299,8 @@ def get_datapreview_recombinant(resource_name, resource_id, owner_org, dataset_t
             priority += 1
         fields.append(out)
 
+    chromo['edit_form'] = h.check_access('resource_update', {'id': resource_id})
```

Comment:
this doesn't look right. `chromo` contains the schema for this recombinant type that is shared by all threads and requests, but `check_access` is returns a value based on the current user's permissions.

Reply:
No longer modifying the `edit_form` chromo value.